nodejs/env

```diff
@@ -0,0 +1,8 @@
+API_URL=e.test.connect.paymentsense.cloud
```

Comment:
Do these need to be passed from here and the docker-compose?

Reply:
This file gets copied into the docker image.

Without it `make build-docker run-docker` would not work as in the `package.json` file there is `export $(cat env | xargs) && node src/index.js`.

Ideally, this file should be named `.env` and the lines in the `docker-compose.yml` should be:

```yaml
...
        CONNECT_E_KEY: ${CONNECT_E_KEY}
        API_URL: ${API_URL}
        WEB_URL: ${WEB_URL}
        PORT: ${PORT}
        GATEWAY_USERNAME: ${GATEWAY_USERNAME}
        GATEWAY_PASSWORD: ${GATEWAY_PASSWORD}
        MERCHANT_URL: ${MERCHANT_URL}
        WEBHOOK_URL: ${WEBHOOK_URL}
...
```

And `docker-compose up --build` will use the values defined in the `.env` file.

What do you think?

Reply:
Yeah that sounds good. Better than having it in multiple places.

Reply:
Thank you. It's resolved in last commit (https://github.com/Paymentsense/connect-e-samples/pull/2/commits/c8abdfb1bc8f3458992fb07e2648c3ce92aa64b6).